### PR TITLE
Add Dockerfile and script for running the tests in CentOS CI

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by tisnik** [comment]
LGTM, thanks. I'm just curious - do we need gcc because some Python modules use its native libraries?

**Comment by msrb** [comment]
> do we need gcc because some Python modules use its native libraries?

Yep, I think `pycrypto` needs to compile some native extension during `pip install`.

**Comment by msrb** [comment]
I got thumbs up from Pavel, merging 😉 Thanks.

File: integration-tests/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM registry.centos.org/centos/centos:7
+MAINTAINER Michal Srb <michal@redhat.com>
```
Comment (jpopelka) [inline integration-tests/Dockerfile]: Jut FYI:  https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

**Pull request merged**